### PR TITLE
[FEATURE] Allow switch to gp3 EBS volumes and migration of existing ones

### PR DESCRIPTION
**Pull request by pinkeen**

### CODE REVIEW

**Comment by pinkeen** [comment]
Ready to merge

**Comment by krisdante** [comment]
Did you test "migration" ? 
What happens when EC2 has gp2 currently?

**Comment by pinkeen** [comment]
Yes I did, migration of existing volumes was one of the primary goals. Existing volumes are "optimized" which takes a few minutes and migrated live without any downtime or performance loss.

File: roles/cs.aws-ebs-vol/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+      {{ aws_ebs_volume_describe_params | to_nice_yaml }}
+      ---------------- Target Parameters  -------------------
+
+      {{ aws_ebs_volume_modify_params | to_nice_yaml }}
+      -------------------------------------------------------
+
+
+- name: Perform volume operations
+  delegate_to: localhost
```
Comment (krisdante) [inline roles/cs.aws-ebs-vol/tasks/main.yml:40]: Did you test it on Jenkins? This "delegate to" is not working well in our CI

**Review by krisdante — APPROVED** [review]

**Pull request merged**